src/main/java/org/jabref/logic/jabmap/MindMapWriter.java

```diff
@@ -0,0 +1,32 @@
+package org.jabref.logic.jabmap;
+
+import org.jabref.gui.JabRefFrame;
+import org.jabref.model.entry.BibEntry;
+
+import java.util.List;
+
+public class MindMapWriter {
+    private static MindMapWriter _instance = null;
+
+    private JabRefFrame jabRefFrame;
+
+    private MindMapWriter() {
+    }
+
+    public MindMapWriter(JabRefFrame jabRefFrame) {
```

Comment:
Logc should not access UI. Bad thing: The whole flow needs to be rethought. -- Reason: JabRef's aim is to work with local variables and not using global ones. `jabRefFrame` is sort of a global variable.